src/GitHub.VisualStudio/UI/Views/PullRequestCreationView.xaml.cs

```diff
@@ -22,8 +22,6 @@ public PullRequestCreationView()
             {
                 d(ViewModel.CancelCommand.Subscribe(_ => NotifyCancel()));
                 d(ViewModel.CreatePullRequest.Subscribe(_ => NotifyDone()));
-                //d(Done.Subscribe(_ => ClearForm()));
```

Comment:
Tried subscribing to the `SimpleViewUserControl`'s Done observable initially to clear the form when a PR is made, which worked but didn't seem like the right approach here. The initial idea was to set `ClearForm()` as the `OnCompleted()` parameter for `CreatePullRequest.Subscribe` but it never reached `ClearForm()`